### ISSUE TITLE
Heroku deployment is broken

### ISSUE THREAD

**Issue by nickmcintyre** [processing/p5.js-web-editor#1721]
Hello! I tried launching a version of the Editor on Heroku yesterday but ran into an error with the (now defunct) mongolab addon.

![mLab error](https://user-images.githubusercontent.com/3719176/103416965-73f68480-4b4e-11eb-9ff6-8f0a5ab48e5d.png)

#### Nature of issue?

- Found a bug

#### Details about the bug: 

- Web browser and version: Firefox 84
- Operating System: macOS Catalina
- Steps to reproduce this bug: Click the big purple button in `developer_docs/deployment.md`.

**Comment by catarak** [comment]
Thanks for reporting!

**Comment by barcar** [comment]
Is this still an issue? It looks like there was a possible fix but I’m seeing the same error?

**Comment by nickmcintyre** [comment]
@barcar please have a look at #1722 and let me know if the updated documentation is easy to follow.

**Comment by DataNeel** [comment]
Just tried deploying to heroku by following the instructions in the documentation, and it still didn't work. I left placeholder values in all of the required fields, and it gets hung up on mongodb.

I looked through the fix that you linked, @nickmcintyre (cc @barcar), but I wasn't totally sure how to interpret it. The official docs still don't mention anything about Atlas.

**Comment by barcar** [comment]
Yes, this is still broken. The pull request with the proposed fixes was never merged.
